src/pysolid/version.py

```diff
@@ -1,63 +0,0 @@
-#!/usr/bin/env python3
-# Author: Zhang Yunjun, Jan 2021
-# Copyright 2020, by the California Institute of Technology.
-
-
-import collections
-import os
-import subprocess
-
-
-###########################################################################
-# release history
-Tag = collections.namedtuple('Tag', 'version date')
-release_history = (
-    Tag('0.2.4', '2022-10-24'),
-    Tag('0.2.3', '2022-10-23'),
-    Tag('0.2.2', '2022-07-20'),
-    Tag('0.2.1', '2022-01-05'),
-    Tag('0.2.0', '2021-11-10'),
-    Tag('0.1.2', '2021-02-24'),
-    Tag('0.1.1', '2021-02-01'),
-    Tag('0.1.0', '2021-01-22'),
-)
```

Comment:
I removed this, but we could add it back, or something like it. I don't really know what value it provides beyond any of these:
* https://github.com/insarlab/MintPy/releases
* https://pypi.org/project/pysolid/#history
* `git log --tags --simplify-by-decoration --pretty="format:%ai %d"`

Reply:
I like the idea of removing the redundant `version.py` file. While checking the version, there seems to have a discrepancy between `git describe --tags` and `setuptools_scm` as below:

```
(pysolid) yunjunz:~/tools/PySolid>$ git describe --tags
v0.2.3-14-g50d06e3
(pysolid) yunjunz:~/tools/PySolid>$ python -m setuptools_scm
0.2.4.dev14+g50d06e3
```

Any idea why?

Reply:
Setting `version_scheme` from `guess-next-dev` to `post-release` solved the inconsistency.